config/defaults.json

```diff
@@ -222,7 +222,7 @@
         },
         "password_recovery": "Forgot your password?",
         "errors": {
-          "invalid_credentials": "Your email or password is incorrect.",
+          "invalid_credentials": "Your email or password are incorrect, please try again.",
```

Comment:
this should still be `is`

`Your email or password is incorrect, please try again.`